src/web/components/ApiKeyManagement/KeyItem.scss

```diff
@@ -0,0 +1,3 @@
+.ApiKeyItem {
```

Comment:
In our usual naming pattern this would be `api-key-item` - do you mind changing this please?

Reply:
This applies to some other class names in this PR as well

Reply:
Have fixed these